massa-proto/proto/massa/api/v1/api.proto

```diff
@@ -318,6 +326,84 @@ message OperationsContext {
   Slot slot = 1;
 }
 
+// GetScExecutionEventsRequest holds request for GetScExecutionEvents
+message GetScExecutionEventsRequest {
+  // Request id
+  string id = 1;
+  // Query
+  GetScExecutionEventsQuery query = 2;
+}
+
+// GetScExecutionEvents Query
+message GetScExecutionEventsQuery {
+  // Filter
+  GetScExecutionEventsFilter filter = 1;
+}
+
+// GetScExecutionEvents Filter
+message GetScExecutionEventsFilter {
+  // Start slot
+  optional Slot start_slot = 1;
+  // End slot
+  optional Slot end_slot = 2;
+  // Caller address
+  optional string caller_address = 3;
+  // Emitter address
+  optional string emitter_address = 4;
+  // Original operation id
+  optional string original_operation_id = 5;
+  // Status
+  repeated ScExecutionEventStatus status = 6;
+}
+
+// GetScExecutionEventsResponse holds response from GetScExecutionEvents
+message GetScExecutionEventsResponse {
+  // Request id
+  string id = 1;
+  // Context
+  GetScExecutionEventsContext context = 2;
+  //ScExecutionEvents wrappers
+  repeated ScExecutionEventWrapper events = 3;
+}
+
+// ScExecutionEvents context
+message GetScExecutionEventsContext {
+  // Slot
+  Slot slot = 1;
+}
+
+// ScExecutionEvent wrapper
+message ScExecutionEventWrapper {
+  // Sc execution context
+  ScExecutionEventContext context = 1;
+  // json data string
+  string data = 2;
+}
+
+// ScExecutionEvent context
+message ScExecutionEventContext {
+  // When was it generated
+  Slot origin_slot = 2;
```

Comment:
why you doesn"t use context ?

Reply:
The context in the top is the query execution context

This context is the SC call execution context

Reply:
Oh yes ok, and this slot is the slot when the sc was generated ?  You can"t use context for that ?

Reply:
ok :)